src/actinia_core/rest/raster_layer.py

```diff
@@ -24,21 +24,26 @@
 """
 Raster layer resources
 """
-from flask import jsonify, make_response
+from flask import jsonify, make_response, request
 from flask_restful_swagger_2 import swagger
+import os
 import pickle
-from .ephemeral_processing import EphemeralProcessing
-from .persistent_processing import PersistentProcessing
+from uuid import uuid4
+from werkzeug.utils import secure_filename
+from actinia_core.rest.ephemeral_processing import EphemeralProcessing
+from actinia_core.rest.persistent_processing import PersistentProcessing
+from actinia_core.rest.map_layer_base import MapLayerRegionResourceBase
 from actinia_core.core.redis_interface import enqueue_job
+from actinia_core.core.exceptions import AsyncProcessError
+from actinia_core.core.utils import allowed_file
 from actinia_core.models.response_models import \
     ProcessingResponseModel, ProcessingErrorResponseModel
-from actinia_core.core.exceptions import AsyncProcessError
-from .map_layer_base import MapLayerRegionResourceBase
+from actinia_core.models.response_models import SimpleResponseModel
 from actinia_core.models.openapi.raster_layer import \
-     RasterInfoResponseModel, RasterRegionCreationModel, RasterInfoModel
```

Comment:
Please also delete the `RasterRegionCreationModel` from the Models when it is not used otherwise anymore.